src/components/ebay-chips-combobox/marko-tag.json

```diff
@@ -5,43 +5,17 @@
         "type": "expression"
     },
     "@html-attributes": "expression",
-    "@borderless": "boolean",
-    "@autocomplete": "string",
     "@expanded": "boolean",
-    "@list-selection": {
-        "enum": ["automatic", "manual"]
-    },
-    "@disabled": "#html-disabled",
-    "@form": "#html-form",
-    "@name": "#html-name",
-    "@placeholder": "#html-placeholder",
-    "@required": "#html-required",
-    "@value": "#html-value",
-    "@role": "never",
-    "@aria-autocomplete": "never",
-    "@aria-haspopup": "never",
-    "@floating-label": "string",
-    "@actionable <actionable>": {
+    "@fluid": "boolean",
+    "@error": "boolean",
+    "@list-selection": "string",
+    "@option <option>": {
```

Comment:
I think this needs to be `"@options <option>[]": {`

See combobox

Reply:
This isn't true for Marko 5 anymore since the new `iterable` changes that were included around the same time as TS was introduced, but it might cause problems in Marko 4. Good catch, I changed it.